lib/rack/response.rb

```diff
@@ -128,6 +128,7 @@ def not_found?;           status == 404;                        end
       def method_not_allowed?;  status == 405;                        end
       def precondition_failed?; status == 412;                        end
       def unprocessable?;       status == 422;                        end
+      def unavailable?;         status == 451;                        end
```

Comment:
We don't have shorthand methods for every status code. Think this is one we'd omit.